lightning/mydump/loader_test.go

```diff
@@ -62,8 +62,8 @@ func (s *testMydumpLoaderSuite) TestLoader(c *C) {
 	}{
 		{name: "i", dataFiles: 1},
 		{name: "report_case_high_risk", dataFiles: 1},
-		{name: "tbl_autoid", dataFiles: 1},
 		{name: "tbl_multi_index", dataFiles: 1},
+		{name: "tbl_autoid", dataFiles: 1},
```

Comment:
It'OK, but seems that is an unnecessary change, what does it affect?

Reply:
Oh the test was broken by #156, seems `tbl_multi_index` is smaller than `tbl_autoid`.

Reply:
Got it.